addons/l10n_it_edi_doi/views/l10n_it_edi_doi_declaration_of_intent_views.xml

```diff
@@ -0,0 +1,140 @@
+                </header>
+                <sheet>
+                    <div class="oe_button_box" name="button_box">
+                        <field name="invoice_ids" invisible="True"/>
+                        <button type="object"
+                                class="oe_stat_button"
+                                name="action_open_invoice_ids"
+                                icon="fa-pencil-square-o"
+                                invisible="not invoice_ids">
+                            <div class="o_field_widget o_stat_info">
+                                <span class="o_stat_text">Invoices</span>
+                            </div>
+                        </button>
+                        <field name="sale_order_ids" invisible="True"/>
+                        <button type="object"
+                                class="oe_stat_button"
+                                name="action_open_sale_order_ids"
+                                icon="fa-pencil-square-o"
+                                invisible="not sale_order_ids">
+                            <div class="o_field_widget o_stat_info">
+                                <span class="o_stat_text">Sale Orders</span>
+                            </div>
+                        </button>
+                    </div>
+                    <group name="main_group">
+                        <group name="left_column">
+                            <field name="partner_id" readonly="state != 'draft'"/>
+                            <field name="company_id" groups="base.group_multi_company" readonly="state != 'draft'"/>
+                            <label for="protocol_number_part1" string="Protocol Number"/>
+                            <div name="protocol_div" class="d-flex">
+                                <field name="protocol_number_part1" readonly="state != 'draft'"/>
+                                <span class="o_form_label mx-3">/</span>
+                                <field name="protocol_number_part2" readonly="state != 'draft'"/>
+                            </div>
+                            <field name="issue_date" readonly="state != 'draft'"/>
+                            <label string="Date Range" for="start_date"/>
+                            <div name="date_range_div" class="d-flex">
+                                <field name="start_date" readonly="state != 'draft'"/>
+                                <span class="o_form_label mx-3"> to </span>
+                                <field name="end_date" readonly="state != 'draft'"/>
+                            </div>
+                        </group>
+                        <group name="right_column">
+                            <div colspan="2" class="o_wrap_label">
+                                <span class="o_form_label">Amounts:</span>
+                            </div>
+                            <field name="currency_id" readonly="state != 'draft'" invisible="True"/>
```

Comment:
Why do we need the `currency_id` in the view?

Reply:
Otherwise the amounts are displayed w/o currency sign (and potentially formatted differently?)